### ISSUE TITLE
Crash after opening expandable menu

### ISSUE THREAD

**Issue by MigRic** [antonpup/Aurora#1690]
Hey guys, so after some problems with my g633, reinstalling my G Hub, I came across a crash .
### Expected Behavior
Normally it should run fine.

### Actual Behavior
It appears to run fine.

### Reproduction steps
After trying to click on those expandable menus, it simply crashes without warning. It also crashes after I left it opened after some seconds.

### Frequency
Every time I try to open an expandable menu.

### Aurora Version:
I've tried v0.6.4b and v0.7.0-rc2, same problem.

### Previously an Issue?
The same version didn't have this kind of problem. I've no idea what happened.

Thanks.

**Comment by Mikowmer** [comment]
What do you mean by "expandable menus"? Do you mean like the Overview, Profiles, Layers, etc? Or when you go through and select layer type?

**Comment by MigRic** [comment]
Sorry if I said badly. I will explain.
For example, if I click on Overview or Profiles or Layers, it'll work fine. But if I click on Layer Type, for example, it will crash. Any menu that is the same thing, like selecting keyboard type, or preferred mouse. Menus that have the "down arrow" that allow you to select different options.

**Comment by MigRic** [comment]
So apparently, it started working again and I have no idea why.
But I have a guess... RTSS wasn't working on CSGO, suddenly it stopped showing OSD. After clearing it's cache, it started working normally again.
Then, for some reason, I started Aurora, just for the sake of it. And voialá, it's working again... I don't know if clearing the cache did it, but I'm happy again =).
I posted this because maybe it can solve someone's problem, I don't know.
